src/lib.rs

```diff
@@ -120,3 +128,42 @@ pub use screenshot::{Image, Screenshot, Video};
 pub use translatable_string::{MarkupTranslatableString, TranslatableList, TranslatableString};
 pub use url;
 pub use xmltree;
+
+cfg_if! {
```

Comment:
I am not a fan of adding cfg_if dependency here.

Reply:
You want the `cfg_if` dependency removed completely?

It would make it very difficult to prevent issues if both the `time` and `chrono` features were enabled. Currently, this just prefers `time` if it's enabled at the same time as `chrono`. This allows the "additive features" concept of Rust to work.

If we get rid of `cfg_if`, we can make a compile error if both features are enabled, but it means we have to remove anywhere that uses `--all-features` (docs, CI, etc) and manually specify features because we can't have all of them enabled without errors.

Reply:
> If we get rid of cfg_if, we can make a compile error if both features are enabled, but it means we have to remove anywhere that uses --all-features (docs, CI, etc) and manually specify features because we can't have all of them enabled without errors.

I would be fine with dropping that yes, in ashpd I have a compile_error if both gtk3/gtk4 features are enabled. We can do the same here